docs/samples/v1beta1/torchserve/transformer.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: serving.kubeflow.org/v1beta1
+kind: InferenceService
+metadata:
+  name: torchserve
+spec:
+  transformer:
+    containers:
+      - image: <username>/image-transformer:latest
+        name: transformer-container
+        env:
+          - name: STORAGE_URI
+            value: pvc://model-pv-claim
```

Comment:
Why transformer needs the actual model?

Reply:
Image transformer uses config.properties in the storage.